plugins/LilyPad/Tooltips.cpp

```diff
@@ -0,0 +1,95 @@
+                   L"Alt-tabbing to another application also makes the cursor visible, but focusing the emulation window hides it again.\n\n"
+                   L"Pressing the button when the mouse is out of focus and visible, it captures the mouse so that it can be used as a controller again.\n\n"
+                   L"Note 1: Though the binding appears on the page of a specific pad, pressing the button affects all mice.\n\n"
+                   L"Note 2: By default PCSX2 uses a double-click by the left mouse button to toggle fullscreen mode, this makes the left mouse button unusable as an input."
+                   L"To disable this option in PCSX2 go to Config > Emulation Settings > GS Window tab, and disable the ''Double-click toggles fullscreen mode'' checkbox.";
+        case ID_ANALOG:
+            return L"Bind a keys that switches the pad from digital mode to analog mode and vice versa.\n\n"
+                   L"This option is useful when analog mode is enabled in a game that does not support it, as this causes the game to not recognise any input or to not even detect a controller.\n\n"
+                   L"This option can also be used to enable analog mode in games that support, but do not automatically enable analog mode.\n\n"
+                   L"Note: Analog mode enables the analog sticks to function on a DualShock controller, while in digital mode it behaves as an original PlayStation controller.\n\n";
+        case ID_IGNORE:
+            return L"Blocks the assigned keyboard button and does not pass it on to PCSX2. It currently cannot block the windows key (for unknown reasons).\n\n"
+                   L"Ignored keys are listed with pad 1's bindings. You do not have to configure ignored keys again when you change the keyboard input mode.";
+        case ID_LOCK_ALL_INPUT:
+            return L"Locks the current state of the pad. Any further input is handled normally, but the initial pad state is the locked state instead of a state with no buttons pressed. "
+                   L"Pressing it again releases the old pad state, if the old pad state had any keys pressed. Otherwise, it's released automatically.";
+        case ID_LOCK_DIRECTION:
+            return L"Locks the current state of the d-pad and analog sticks. Pressing this when all input is locked unlocks only the pad and sticks."
+                   L"Pressing it again will lock them again, keeping the buttons locked.";
+        case ID_LOCK_BUTTONS:
+            return L"Locks the current state of the buttons. Pressing this when all input is locked unlocks only the buttons. "
+                   L"Pressing it again will lock them again, keeping the d-pad and analog sticks locked.";
+        case IDC_TURBO:
+            return L"Automatically presses/depresses the input every other time the button is polled.";
```

Comment:
Yeah. Come to think of it I actually got several of these messages from ChickenLiver's original LilyPad documentation and comments. So perhaps his name should be in the copyright after all.